design/ceph/object/zone-group.md

```diff
@@ -53,17 +54,36 @@ The order in which these resources are created is not important.
 1. The realm named in the `realm` section must be the same as the ceph-object-realm resource the zone group is a part of.
 3. When resource is deleted, zone group are not deleted from the cluster. Zone group deletion must be done through toolboxes.
 
+#### Hostname Configuration
+
+ A list of hostnames to use for accessing the bucket directly like a subdomain in the endpoint. For example if the default service endpoint `rook-ceph-rgw-my-store.rook-ceph.svc:80` added and the object store contains a bucket named `sample`, then the s3 style dns domain would be `http://sample.rook-ceph-rgw-my-store.rook-ceph.svc:80`. More details can about the feature can be found in [IBM Documentation](https://www.ibm.com/docs/en/storage-ceph/6?topic=configuration-add-wildcard-dns).
```

Comment:
Blaine and I have so many questions not answered in [this thread](https://github.com/rook/rook/pull/13022#discussion_r1372254142). This paragraph does not give enough details to understand the feature. What exactly will the Rook user need to configure to get the DNS subdomain style names?

Reply:
For upstream rook we need to link to upstream ceph docs. The IBM doc for v6 is downstream.

Reply:
I would tend to agree. It's a bad look for the Ceph project if features exist and aren't documented upstream.

Reply:
> Blaine and I have so many questions not answered in [this thread](https://github.com/rook/rook/pull/13022#discussion_r1372254142). This paragraph does not give enough details to understand the feature. What exactly will the Rook user need to configure to get the DNS subdomain style names?

The [Line74](https://github.com/rook/rook/pull/13022/files#diff-5a34ba9d68aa04d4c6498efcb96f5a8256a5db44b244c41a4e5094407d22f544R74) mentions about it briefly, kindly let me know what I am missing there. Rook enables this feature from RGW point of view. Now user to configure DNS settings in k8s cluster to access the bucket via url